homeassistant/components/webhook/__init__.py

```diff
@@ -49,7 +57,7 @@ def async_register(
     """Register a webhook."""
     handlers = hass.data.setdefault(DOMAIN, {})
 
-    if webhook_id in handlers:
+    if is_registered(hass, webhook_id):
```

Comment:
Maybe not use this internally, since there's an extra dict created every time `is_registered` is called.